fullstack-cert/html-projects/framed-videos/styles.css

```diff
@@ -0,0 +1,35 @@
+* {
```

Comment:
remember this prototype will be in the HTML chapter. 
so no CSS will be involved here